tests/tests.js

```diff
@@ -536,3 +532,48 @@ QUnit.test('escapedCharacterString', function(assert) {
     'hello &amp;amp;&amp;gt;&amp;#39;&amp;lt;&amp;amp; &amp;quot;'
   );
 });
+
+QUnit.test('unescapeHTML', function(assert) {
+  var html = '&amp;lt;div&amp;gt;&amp;lt;/div&amp;gt;';
+  var unescapedHTML = unescapeHTML(html, 2);
+  assert.equal(unescapedHTML, '<div></div>');
+});
+
+QUnit.test('minimizeStyles', function(assert) {
+  var message = {
+    'html': [
+        '<div id="myId"',
+        'style="animation-delay: 0s; width: 5px;" ',
+        '></div>'
+    ],
+    'frameHoles': null,
+    'idToStyleIndex': {"myId": 1},
+    'windowHeight': 5,
+    'windowWidth': 5,
+    'frameIndex': '0'
+  };
+  minimizeStyles(message);
+  assert.equal(message.html[1], 'style="width: 5px;" ');
+});
+
+QUnit.test('serialize tree: end-to-end', function(assert) {
```

Comment:
I created an additional test, similar to the one above, but with those three lines added.  The resulting style attribute is relatively large.  Let me know if the test I added is what you had in mind.

Also, remind me about the transform-origin behavior?  Why does this attribute change the transform origin?  Is it because the border changes the center point of the element?